scan/lib/scan/options.rb

```diff
@@ -292,7 +297,19 @@ def self.available_options
                                      deprecated: "Use `--output_files` instead",
                                      conflicting_options: [:output_files],
                                      optional: true,
-                                     is_string: true)
+                                     is_string: true),
+        FastlaneCore::ConfigItem.new(key: :app_identifier,
+                                     env_name: 'SCAN_APP_IDENTIFIER',
+                                     optional: true,
+                                     description: "The bundle identifier of the app to uninstall (only needed when enabling reinstall_app)",
+                                     code_gen_sensitive: true,
+                                     default_value: ENV["SCAN_APP_IDENTITIFER"] || CredentialsManager::AppfileConfig.try_fetch_value(:app_identifier),
```

Comment:
Is `ENV["SCAN_APP_IDENTITIFER"]` spelled wrong and in `default_value` for a specific reason? 🤔

Reply:
Because copy paste :)) I copied it from here https://github.com/fastlane/fastlane/blob/2d08ade8e28670b49b1f8abfe6121affb4905092/snapshot/lib/snapshot/options.rb#L125 I'll send another PR to fix that.

Reply:
Done: https://github.com/fastlane/fastlane/pull/13093